### ISSUE TITLE
Setting proxy in Incognito mode has no effect

### ISSUE THREAD

**Issue by wang0916** [kaliiiiiiiiii/Selenium-Driverless#294]
I tried to use a proxy in Incognito mode, but it didn't work
```python
from selenium_driverless import webdriver
import asyncio


async def main():
    proxy = "http://127.0.0.1:10809"
    options = webdriver.ChromeOptions()
    options.add_argument('--incognito')
    async with webdriver.Chrome(options=options) as driver:
        # this will overwrite the proxy for ALL CONTEXTS
        await driver.set_single_proxy(proxy)
        await driver.get("https://ip.900cha.com/")
        await asyncio.sleep(1000)

asyncio.run(main())
```

**Comment by ttlns** [comment]
> I tried to use a proxy in Incognito mode, but it didn't work ` from selenium_driverless import webdriver import asyncio
> 
> async def main(): proxy = "http://127.0.0.1:10809" options = webdriver.ChromeOptions() options.add_argument('--incognito') async with webdriver.Chrome(options=options) as driver: # this will overwrite the proxy for ALL CONTEXTS await driver.set_single_proxy(proxy) await driver.get("https://ip.900cha.com/") await asyncio.sleep(1000)
> 
> asyncio.run(main())

Yes, because for setting proxies dynamically, a chrome-extension is required
Adding those chrome extensions is only really possible without incognito

Workaround: Startup without incognito, use https://kaliiiiiiiiii.github.io/Selenium-Driverless/api/Chrome.html#selenium_driverless.webdriver.Chrome.new_context 
Should be aequivalent
